src/main/java/Duke.java

```diff
@@ -1,10 +1,105 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static void printLines() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printLinesWithText(String response) {
+        //Prints horizontal lines in between response
+        printLines();
+        System.out.println(response);
+        printLines();
+        System.out.print(System.lineSeparator());
```

Comment:
I like how you did this to simplify your print statements

Reply:
Thanks!